app/models/contestant.rb

```diff
@@ -1,3 +1,9 @@
 class Contestant < ApplicationRecord
   validates_presence_of :name, :age, :hometown, :years_of_experience
+  has_many :contestant_projects
+  has_many :projects, through: :contestant_projects
+
+  def add_project(project)
```

Comment:
See my message in your `Contestant` controller. With this method you are essentially adding a project to a contestant. This can be achieved and create a more RESTful app by creating a new relationship between a contestant and a project.